.vib/odoo/cypress/cypress/integration/odoo_spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+import 'cypress-real-events/support';
+
+it('allows creating a company', () => {
+  cy.login();
+  cy.get('[title="Home Menu"]').click();
+  cy.contains('Settings').click();
+  cy.contains('Users & Companies').click();
+  cy.get('[data-menu-xmlid="base.menu_action_res_company_form"]').click();
+  cy.get('.o_list_button_add').click();
+  cy.fixture('companies').then((company) => {
+    cy.get('[name="name"]').type(`${company.newCompany.name}.${random}`);
+    cy.contains('Save').click();
+    cy.contains('.oe_title', 'Company Name');
+    cy.contains('.oe_title', `${company.newCompany.name}.${random}`);
+  });
+});
+
+it('allows installing/uninstalling an application', () => {
+  cy.login();
+  cy.get('[title="Home Menu"]').click();
+  cy.contains('[data-menu-xmlid="base.menu_management"]', 'Apps').click();
+  cy.get('[title="Sales"]').within(() => {
+    cy.contains('Install').click();
+    cy.get('.o_blockUI').should('not.exist');
```

Comment:
This is a loader that appears after I click on installing the application. 
I want to make sure it's gone so I check for it in **two** places knowingly. 
The installation is slow so sometimes the loader doesn't appear at this step, but later.